cmd/influxd/launcher/launcher.go

```diff
@@ -300,9 +335,17 @@ func (m *Launcher) run(ctx context.Context, opts *InfluxdOpts) (err error) {
 		return err
 	}
 
-	// apply migrations to metadata store
-	if err := migrator.Up(ctx); err != nil {
-		m.log.Error("Failed to apply migrations", zap.Error(err))
+	// apply migrations to the bolt metadata store
+	if err := boltMigrator.Up(ctx); err != nil {
+		m.log.Error("Failed to apply bolt migrations", zap.Error(err))
+		return err
+	}
+
+	sqliteMigrator := sqlite.NewMigrator(m.sqlStore, m.log.With(zap.String("service", "sqlite migrations")))
+
+	// apply migrations to the sqlite metadata store
+	if err := sqliteMigrator.Up(&sqliteMigrations.All{}); err != nil {
```

Comment:
The `Up` method takes an argument for a migration "source", which I'm just using to allow a `go-bindata` source to be injected during testing. The `sqliteMigrations.All{}` struct has methods to satisfy the `MigrationSource` interface which is shown in a file further down. This is all pretty cumbersome - maybe there's a more elegant way to do it. It could be simplified significantly using the `go1.16` native `go:embed`